src/tunserver.cpp

```diff
@@ -1906,6 +1913,14 @@ void c_tunserver::set_remove_peer_tometout(unsigned int timeout_seconds) {
 	peer_timeout = std::chrono::seconds(timeout_seconds);
 }
 
+void c_tunserver::set_prefix_len(int prefix) {
+	m_prefix_len = prefix;
+}
+
+void c_tunserver::set_prefix(const string &prefix) {
+	m_ipv6_prefix = prefix;
```

Comment:
@robertoleksy this var naming is not consistent, imo better

m_ipv6_prefix_addr
m_ipv6_prefix_len 

please apply even when this is merged